### DISCUSSION TITLE
How do I remove a collection from Directus but not the DB? (Revert to "Database Only")

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#11140] [Archive]
When I load up Directus with an existing database, all the collections are listed as "Database Only". After configuring one of them in Directus, this of course goes away. How would I go about removing that collection from Directus again without removing it from the database? Essentially, how do I reverse the collection back to it's original state where it exists in the database but is not configured in Directus, and therefore is listed as "Database Only"?

**Comment by rijkvanzanten** [comment]
There's no way to do that in Directus itself right now. You could delete the row from directus_collections or directus_fields manually in the database though.

That being said, the configuration stored in those rows is all progressive enhancements on top of the underlying table. What's the use case of reverting it back to database-only? 🤔

**Reply by AaronBeaudoin** [reply]
I was experimenting with displaying some database tables that I decided I don't actually want to manage in Directus right now, so I wanted to "reset" them so I could start from scratch adding them back in later if I ever decide to.

I did find the two tables `directus_collections` and `directus_fields` and the relevant rows, but I wasn't sure whether there was anything else that I would need to remove to revert things back to "Database Only".

**Reply by rijkvanzanten** [reply]
Gotcha, there's nothing else to do 👍🏻 (there's also no real difference between having it "database only" or not, as the difference exclusively lives in the metadata of the internal services)

**Reply by joselcvarela** [reply]
For visibility purposes, you can also experiment Roles and Permissions 👍
